book/installation.rst

```diff
@@ -98,9 +98,6 @@ optional second argument of the ``new`` command:
     $ symfony new my_project_name 2.3.26
     $ symfony new my_project_name 2.6.5
 
-    # use the most recent LTS (Long Term Support) version
-    $ symfony new my_project_name lts
```

Comment:
Yes. In GitHub's split view it's easier to understand why I removed it (the explanation is duplicated!)

![lts_repeated_explanation](https://cloud.githubusercontent.com/assets/73419/9299996/d0d6c104-44c1-11e5-8e7f-7aa6b808ce4b.png)